README.md

```diff
@@ -75,9 +76,26 @@ scrape_configs:
 
 `honor_labels: true` is important because exporter returns metrics with `instance` label set.
```

Comment:
`honor_labels: true` is important because **the** exporter returns metrics with `instance` label set.
<-- not very clear what this means

Reply:
It is explanation for Prometheus configuration https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config